### PR TITLE
Remove bitbucket pipelines security group from dev

### PR DESCRIPTION
**Pull request by xmunoz**
This security group only added additional whitelist entries to port 22,
and the Development security group has now been modified to whitelist
everyone on port 22.

Also, bump size of the attached block storage for dev, because why not.

### CODE REVIEW

File: images/backend.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "hostname": "{{env `PERM_ENV` }}",
     "instance_type": "m4.large",
-    "volume_size": "100",
+    "volume_size": "1000",
```
Comment (jasonaowen) [inline images/backend.json:4]: Ideally, this change would be in a separate commit, since it's unrelated to the security group changes.

File: instances/dev.tf [hunk 2]
```diff
@@ -46,14 +46,10 @@ data "aws_security_group" "dev_sg" {
   name = "Development"
```
Comment (jasonaowen) [inline instances/dev.tf:46]: > the Development security group has now been modified to whitelist everyone on port 22

Just to double-check my understanding: the `dev_sg` security group is not managed by terraform, and so that modification was made in the AWS web UI and is not tracked by this repo; is that correct?

**Review by jasonaowen — APPROVED** [review]
I confirmed that the bitbucket pipeline security group only granted port 22 access, and that the dev security group grants port 22 access to the world:

Screenshot 2020-11-30 14:14:46

So I think this change makes sense to me!

File: instances/dev.tf (hunk 2 above)
Comment (xmunoz) [reply instances/dev.tf:46]: Terraform will only manage EC2 instances, AMIs, and (soon) load balancers and autoscaling groups (used by future staging + prod).
